viaf.py

```diff
@@ -709,8 +721,15 @@ def process_contributor(self, contributor):
                 contributor.viaf, contributor.sort_name, contributor.display_name
             )
         else:
+            known_titles = {}
```

Comment:
I feel like this should be a set, rather than a dictionary whose keys are set to 1 (which is basically a set that's harder to read).